base/boot.jl

```diff
@@ -195,60 +195,61 @@ end
 Expr(@nospecialize args...) = _expr(args...)
 
 abstract type Exception end
-mutable struct ErrorException <: Exception
-    msg::AbstractString
+struct ErrorException <: Exception
+    msg::String
     ErrorException(msg::AbstractString) = new(msg)
 end
 
 macro _noinline_meta()
     Expr(:meta, :noinline)
 end
 
-struct BoundsError        <: Exception
+struct BoundsError <: Exception
     a::Any
     i::Any
     BoundsError() = new()
     BoundsError(@nospecialize(a)) = (@_noinline_meta; new(a))
     BoundsError(@nospecialize(a), i) = (@_noinline_meta; new(a,i))
 end
-struct DivideError        <: Exception end
-struct OutOfMemoryError   <: Exception end
-struct ReadOnlyMemoryError<: Exception end
-struct SegmentationFault  <: Exception end
-struct StackOverflowError <: Exception end
-struct UndefRefError      <: Exception end
-struct UndefVarError      <: Exception
+struct DivideError         <: Exception end
+struct OutOfMemoryError    <: Exception end
+struct ReadOnlyMemoryError <: Exception end
+struct SegmentationFault   <: Exception end
+struct StackOverflowError  <: Exception end
+struct UndefRefError       <: Exception end
+struct UndefVarError <: Exception
```

Comment:
Why remove the alignment just on this one?

Reply:
See the following line, this one is defined in full